api/v1alpha1/serviceexport_types.go

```diff
@@ -9,6 +9,27 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// ExportedAnnotationsWeight is the key for the weight annotation as an exportedAnnotation on a ServiceExport.
+	ExportedAnnotationsWeight = "weight"
+)
+
+// ServiceExportSpec describes an exported service extra information.
+type ServiceExportSpec struct {
+	// exportedLabels describes the labels exported.
+	// +optional
+	ExportedLabels map[string]string `json:"exportedLabels,omitempty"`
+	// exportedAnnotations describes the annotations exported.
+	// Possible Annotations: "weight".
+	// "weight" specifies the proportion of requests forwarded to the cluster within a serviceImport.
+	// This is computed as weight/(sum of all weights in the serviceImport).
+	// If weight is set to 0, no traffic should be forwarded for this entry. If unspecified, weight defaults to 1.
```

Comment:
so we decide to not support the less than average traffic to a cluster while split between the rest case?